src/frontend/src/pages/PPQPage/PPQPage.tsx

```diff
@@ -9,9 +12,70 @@ interface IComponentProps {
   enablePiaIntakeForm: boolean;
 }
 function PPQLandingPage(props: IComponentProps) {
-  // TODO For ticket 402 use pia value to decide render pia intake or not
   const { enablePiaIntakeForm } = props;
-  return (
+  return enablePiaIntakeForm ? (
+    <div className="ppq-container background">
+      <div className="ppq-section results-wrapper">
```

Comment:
Another alternative would be to not define such specific classes unless absolutely necessary.

Ideally, we should be able to cover 90-95% of use cases with generic global classes / nested selectors as this will provide the most consistent look and feel for the application.

Reply:
i agree we do not define such specific classes unless we must do it.

Reply:
sure, I'd love that! No unnecessary classes, unless absolutely required.

so, I think we are action pending here @binggaofw, right? I mean if you are getting rid of the unnecessary classes @binggaofw or **if we** absolutely require it, can you rename it appropriately?